src/addons.c

```diff
@@ -76,7 +76,9 @@ static void menu_select_cb(MenuLayer *menu_layer, MenuIndex *cell_index, BasicMe
 
 static void initialize_ui(void) {
   s_window = window_create();
+#ifdef PBL_SDK_2
   window_set_fullscreen(s_window, false);
```

Comment:
window_set_fullscreen is no longer available in SDK 3.0
I don't know if there should be a statusbar, but for SDK 3.0 we should be using StatusBarLayer API